### DISCUSSION TITLE
What orange and blue represent?

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1097] [Q&A] [answered]
I know this have been told in the course but I am not able to recall.
Please help me with this.
**Why these blocks are orange and blue ?**
![image](https://user-images.githubusercontent.com/91677627/154786548-639fe64c-f834-448a-b1f8-c7f198f69912.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Noveleader 
This is something related to REMIX IDE meant to help you to identify how each one will interact with the contract
The orange ones create  change on blockchain, which means is going to trigger a new transaction and you will have to pay gas in or order to complete it.
The blue ones are just view ones, means that you are just getting information from the blockchain but without asking to create a new register, so they are gas free.

**Reply by Noveleader** [reply]
Thanks, was missing the same point.
